backend/census_historical_migration/workbooklib/end_to_end_core.py

```diff
@@ -297,7 +298,7 @@ def run_end_to_end(user, audit_header, result):
             audit_header.DBKEY,
             sac.report_id,
             result["transformations"],
-            )
+        )
```

Comment:
Another place were we could reset the result fields before processing the next submission.

Reply:
We may want to reset here regardless of previous reset to ensure the last submission processed does not leave unnecessary data in memory

Reply:
I'm not opposed but right now the commands use `result` to print stuff after it's done running. For now I'm just resetting at the start.